com.unity.multiplayer.mlapi/Editor/CodeGen/CodeGenHelpers.cs

```diff
@@ -12,6 +12,14 @@
 using Unity.CompilationPipeline.Common.ILPostProcessing;
 using UnityEngine;
 
+#if UNITY_2021_1_OR_NEWER
+#warning Built-in Unity ILPP needs to be re-enabled after fixing self-referencing assembly reload issue on 2021.1+ and trunk
```

Comment:
I'd like to highlight this and explain here a little further.

As @wackoisgod mentioned in the PR description, we want to re-enable Unity's built-in ILPP instead of this workaround:
> The goal is that we can re-enable the unity version once self-references are fixed and other issues are resolved.

However, this workaround is still needed for 2019.4 target but 2020.x(LTS)+ will be OK without the workaround implementation and just work fine with Unity ILPP.
Also to overstate a little, we expect reloading self-referencing assemblies to be fixed on 2021.x+ and trunk in the future, hence this compiler warning on 2021.1+